### PR TITLE
Adjust styles

### PR DESCRIPTION
**Pull request by ltclmbdev**
<details>
  <summary>Short (up to 60 chars) title:</summary>
  
</details>
<br/>
<details>
  <summary>Medium (60-80 chars) title:</summary>
  
</details>
<br/>
<details>
  <summary>Long (more than 80 chars) title:</summary>
  
</details>
<br/><br/>
(you can see used font size as red number to the right from instructor's name)
<br/><br/>
P.S. I did some small research and found that our longest title contains 85 chars and average length title contains 39 chars. So current font size "breakpoints" are: 
> 60 chars - fontSize: 60px
> 60 && < 80 chars - fontSize: 56px
< 80 chars - fontSize: 52px

### CODE REVIEW

File: src/chromium.ts [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-    await page.goto(url);
-    const file = await page.screenshot({ type });
-    return file;
+export async function getScreenshot(
+  url: string,
+  type: FileType,
+  isDev: boolean
+) {
+  const page = await getPage(isDev)
+  await page.setViewport({ width: 1200, height: 685 })
```
Comment (ltclmbdev) [inline src/chromium.ts:21]: decrease sizes of output image

File: src/parser.ts [hunk 2]
```diff
@@ -1,54 +1,63 @@
...
-        heights: getArray(heights),
-    };
-    parsedRequest.images = getDefaultImages(parsedRequest.images, parsedRequest.theme);
-    return parsedRequest;
+  const parsedRequest: ParsedRequest = {
+    fileType: extension === "jpeg" ? extension : "png",
+    text: decodeURIComponent(text),
+    theme: theme === "dark" ? "dark" : "light",
+    md: md === "1" || md === "true",
+    fontSize: fontSize || "60px",
```
Comment (ltclmbdev) [inline src/parser.ts:34]: set default font equal 60px

File: src/template.ts [hunk 3]
```diff
@@ -110,57 +127,71 @@ function getCss(theme: string, fontSize: string) {
...
-    const { square_cover_large_url, title, instructor } = course
-    const images = [square_cover_large_url]
-    const text = title
-
-    return `<!DOCTYPE html>
+  const { theme, md, fontSize, widths, heights } = parsedReq
+  const { square_cover_large_url, title, instructor } = course
+  const images = [square_cover_large_url]
+  const text = title
+  const adjustedFontSize =
```
Comment (ltclmbdev) [inline src/template.ts:140]: define needed font size depending on title length

File: src/template.ts [hunk 4]
```diff
@@ -110,57 +127,71 @@ function getCss(theme: string, fontSize: string) {
...
-                    <div class="divider"></div>
-                    <div class="with-author-holder">
-                        <div>with</div>
-                        <div class="author-name">${emojify(
-                md ? marked(instructor.full_name) : sanitizeHtml(instructor.full_name)
-            )}</div>
-                    </div>
+        <div class="wrapper">
+            ${getImage(
+              "https://d2ddoduugvun08.cloudfront.net/items/0s0k2J0j3i3O2m0D1k2K/eggo.svg",
```
Comment (ltclmbdev) [inline src/template.ts:154]: hardcoded link to eggo for now, need to be changed to the one hosted on our side (I failed to find the way to use image placed locally in the project)

File: src/chromium.ts (hunk 1 above)
Comment (joelhooks) [reply src/chromium.ts:21]: Why reduce it?

**Pull request merged**